tests/integration/simple-app.test.ts

```diff
@@ -210,6 +243,39 @@ test<FixtureTestContext>('cacheable route handler is cached on cdn (revalidate=f
   )
 })
 
+test<FixtureTestContext>('purge API is not used when unstable_cache cache entry gets stale', async (ctx) => {
+  await createFixture('simple', ctx)
+  await runPlugin(ctx)
+
+  // set the NETLIFY_PURGE_API_TOKEN to get pass token check and allow fetch call to be made
+  vi.stubEnv('NETLIFY_PURGE_API_TOKEN', 'mock')
+
+  const page1 = await invokeFunction(ctx, {
+    url: '/unstable_cache',
+  })
+  const data1 = load(page1.body)('pre').text()
+
+  // allow for cache entry to get stale
+  await new Promise((res) => setTimeout(res, 2000))
```

Comment:
is mocking and advancing the clock here problematic?

Reply:
we use above in plenty of other places that check behaviors related to stale cache items so I just the same method here

I'm not sure if attempting to mock time would work